packages/node_modules/@webex/react-component-utils/src/activity.js

```diff
@@ -10,11 +12,23 @@ import {isImage} from './files';
  * @param {object} conversation
  * @param {object} activityObject
  * @param {any} actor
+ * @param {boolean} [sanitize = false]
  * @returns {object}
  */
-export function constructActivity(conversation, activityObject, actor) {
+export function constructActivity(conversation, activityObject, actor, sanitize = false) {
   const clientTempId = `sdk-widget-${uuid.v4()}`;
 
+  const modifiedActivityObject = Object.assign({}, activityObject);
+
+  if (sanitize) {
+    if (modifiedActivityObject.displayName) {
+      modifiedActivityObject.displayName = DOMPurify.sanitize(activityObject.displayName);
+    }
+    if (modifiedActivityObject.content) {
+      modifiedActivityObject.content = DOMPurify.sanitize(activityObject.content);
```

Comment:
Instead of using `DOMPurify` here, you should use `escape` from `@webex/helper-html` when the object is being created.

Reply:
...something something i also expect tests 😈